includes/Modules/Sign_In_With_Google.php

```diff
@@ -117,9 +118,10 @@ public function __construct(
 		Assets $assets = null
 	) {
 		parent::__construct( $context, $options, $user_options, $authentication, $assets );
+
 		$this->existing_client_id        = new Existing_Client_ID( $this->options );
+		$this->is_woocommerce_active     = $this->check_is_woocommerce_active();
```

Comment:
This check must happen only on demand. We don't need to check whether the WooCommerce plugin is active for every request.